sbin/build.sh

```diff
@@ -860,6 +860,8 @@ generateSBoM() {
     return
   fi
 
+  # exit from $(setupAntEnv) is not propagated. We ahve to ensure it is fatal, co calling it one more times
```

Comment:
```suggestion
  # exit from $(setupAntEnv) is not propagated. We have to ensure it runs, so calling it one more time
```

Reply:
This do not seem right. The patch is ensuring, that the `exit 2` is propagated, thus really making the call to `setupAntEnv` fatal, if it fails

Reply:
Argh, sorry, I committed that change, please do override it with your explanation

Reply:
ty, updated!